CondFormats/PhysicsToolsObjects/test/SiStripDeDx3DBuilder.cc

```diff
@@ -24,7 +24,7 @@ class SiStripDeDx3DBuilder : public edm::EDAnalyzer {
   virtual void analyze(const edm::Event&, const edm::EventSetup&);
 
 private:
-  edm::FileInPath fp_;
+  //edm::FileInPath fp_;
```

Comment:
NOTE: this was left intentionally commented out as the code that uses fp_ was already commented out.